src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -2434,5 +2438,53 @@ class Program
     int [|a|];
 }");
         }
+
+        [WorkItem(40053, "https://github.com/dotnet/roslyn/issues/40053")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
+        public async Task TestEqualsNullableAnnotation()
+        {
+            await TestWithPickMembersDialogAsync(
```

Comment:
does this need to go through the 'pick members dialog' test path?  this doesn't feel dialog specific.

Reply:
This [code](http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider.cs,253) indicates that I cannot test "generate operators" without having a dialog, then I think "pick members dialog" is necessary

Reply:
you could definitely update that code :)  just add a default `generateOperators` to http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider.cs,239 and pass that along.  No tests need to change, but you can now pass `generateOperators: true` from your test.

Reply:
What benefit is there in changing the current behavior? Current code design is that a dialog is required to allow generation of operators. If we change the code, that distinction becomes harder to follow. Testing with a mocked out dialog is not an issue

Reply:
@CyrusNajmabadi this seems to be the only outstanding issue. I'm approving as I don't see this blocking, but want to wait for your feedback before merging.